frontend/cypress/tests/chartView/chartView_zoomFeature.cy.js

```diff
@@ -39,3 +39,260 @@ describe('zoom features in code view', () => {
+  });
+
+  it('changing \'since\' date again will result in a different zoom view', () => {
+    cy.get('div.mui-textfield.search_box > input:visible')
+      .should('be.visible')
+      .type('yong hao');
+
+    cy.get('input[name="since"]:visible')
+      .type('2018-05-20');
+
+    cy.get('.icon-button.fa-list-ul')
+      .should('be.visible')
+      .first()
+      .click();
+
+    cy.get('#tab-zoom')
+      .should('be.visible');
+
+    cy.get('#tab-zoom .ramp .ramp__slice')
+      .last()
+      .invoke('attr', 'title')
+      .should('eq', '[2018-05-20] Apply single responsibility principle to frontend component (#99): +201 -90 lines ');
+  });
+
+  it('changing the \'until\' date will results in a different zoom view', () => {
+    cy.get('div.mui-textfield.search_box > input:visible')
+      .should('be.visible')
+      .type('eugene');
+
+    cy.get('input[name="until"]:visible')
+      .type('2019-08-19');
+
+    cy.get('.icon-button.fa-list-ul')
+      .should('be.visible')
+      .first()
+      .click();
+
+    cy.get('#tab-zoom')
+      .should('be.visible');
+
+    cy.get('#tab-zoom .ramp .ramp__slice')
+      .first()
+      .invoke('attr', 'title')
+      .should('eq', '[2019-08-18] AboutUs: update team members (#867): +94 -12 lines ');
+  });
+
+  it('changing the \'until\' date again will result in a different zoom view', () => {
+    cy.get('div.mui-textfield.search_box > input:visible')
+      .should('be.visible')
+      .type('metta');
+
+    cy.get('input[name="until"]:visible')
+      .type('2018-07-20');
+
+    cy.get('.icon-button.fa-list-ul')
+      .should('be.visible')
+      .first()
+      .click();
+
+    cy.get('#tab-zoom')
+      .should('be.visible');
+
+    cy.get('#tab-zoom .ramp .ramp__slice')
+      .first()
+      .invoke('attr', 'title')
+      .should('eq', '[2018-07-20] Dashboard: remove navigation bar (#171): +5 -1 lines ');
+  });
+
+  it('changing the \'until\' and \'since\' date again will result in a different zoom view', () => {
+    cy.get('div.mui-textfield.search_box > input:visible')
+      .should('be.visible')
+      .type('eugene');
+
+    cy.get('input[name="since"]:visible')
+      .type('2018-08-27');
+
+    cy.get('input[name="until"]:visible')
+      .type('2019-03-09');
+
+    cy.get('.icon-button.fa-list-ul')
+      .should('be.visible')
+      .first()
+      .click();
+
+    cy.get('#tab-zoom')
+      .should('be.visible');
+
+    cy.get('#tab-zoom .ramp .ramp__slice')
+      .first()
+      .invoke('attr', 'title')
+      .should('eq', '[2019-03-08] [#587] Fix unoriented output messages (#593): +105 -69 lines ');
+
+    cy.get('#tab-zoom .ramp .ramp__slice')
+      .last()
+      .invoke('attr', 'title')
+      .should('eq', '[2018-08-27] [#311] code view: differentiate untouched code more (#314): +12 -5 lines ');
+  });
+
+  it('changing the \'until\' and \'since\' date again will result in a different zoom view', () => {
```

Comment:
Could the name of these two tests be more specific in what they are testing so that they aren't identical?